packages/terra-functional-testing/src/commands/axe/run.js

```diff
@@ -43,7 +47,7 @@ const runAxe = (overrides = {}) => {
     axe.run(document, opts, function (error, result) {
       done({ error, result });
     });
-  }, { rules, restoreScroll: true, runOnly: ['wcag2a', 'wcag2aa', 'wcag21aa', 'section508'] });
```

Comment:
`restoreScroll` was removed: https://github.com/dequelabs/axe-core/issues/2388

Reply:
I didn't know the scrolling that it used to do was _only_ for color-contrast.